### PR TITLE
update cat command

### PR DESCRIPTION
**Pull request by andyxning**
* fix typo
* add output line numbers

### CODE REVIEW

File: pages/common/cat.md [hunk 1]
```diff
@@ -10,6 +10,14 @@
 
 `cat {{file1}} {{file2}} > {{target-file}}`
 
-- Append serveral files into the target file:
+- Append several files into the target file:
 
 `cat {{file1}} {{file2}} >> {{target-file}}`
+
+- Number the non-blank lines:
```
Comment (rubenvereecken) [inline pages/common/cat.md]: Sounds like specialized behavior of the one below, wouldn't switching make more sense? Also I think you can drop "the".

Comment (andyxning) [inline pages/common/cat.md]: both `linux` and `osx` have the `-b` and `-n` options for `cat`.

Comment (rubenvereecken) [inline pages/common/cat.md]: I don't mean that, I just thought `-n` is more common than `-b` (a specialized version of `-n` you could say).

Comment (andyxning) [inline pages/common/cat.md]: you mean that we may only need `-n`?

**Comment by rubenvereecken** [comment]
Alright, I'm happy with this one :+1:

**Comment by waldyrious** [comment]
LGTM, thanks!

**Pull request merged**